src/hooks/usePrevious.js

```diff
@@ -0,0 +1,9 @@
+import { useRef, useEffect } from 'react';
+
+export const usePrevious = (value) => {
```

Comment:
Hmm, is this being used somewhere in the code? :thinking:

Reply:
Good catch! It was, but after another refactoring it's not. Going to remove it 😊